segmentstore/storage/impl/src/main/java/io/pravega/segmentstore/storage/impl/hdfs/HDFSStorage.java

```diff
@@ -187,7 +189,7 @@ private boolean isSealed(Path path) throws FileNameFormatException {
         return getEpochFromPath(path) == MAX_EPOCH;
     }
 
-    protected FileSystem openFileSystem(Configuration conf) throws IOException {
+    FileSystem openFileSystem(Configuration conf) throws IOException {
```

Comment:
Any reason why we can't make this private?

Reply:
This is overridden by the test(`HDFSStorageTest`).